content/reference/endpoints/bathrooms.md

```diff
@@ -0,0 +1,95 @@
+# Bathrooms
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+  <li><a href="#rentals_write" role="tab" data-toggle="pill">rentals_write</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name                | Type    | Read/Write | Description
+--------------------|---------|------------|------------
+id                  | Integer | Read       | Bathroom's id.
+--------------------|---------|------------|------------
+name                | [Object](/reference/enums#formats)| Read       | Bathroom's title, list of locales described in [enums section](/reference/enums#locales).
+bath                | Integer | Read       | Bathroom's number of baths.
+shower              | Integer | Read       | Bathroom's number of showers.
+wc                  | Integer | Read       | Bathroom's number of wc's.
+--------------------|---------|------------|------------
+created_at          | [Time](/reference/enums#formats) | Read        | Bathroom's create time.
+updated_at          | [Time](/reference/enums#formats) | Read        | Bathroom's update time.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="rentals_write" markdown="1">
+Name                | Type    | Read/Write | Description
+--------------------|---------|------------|------------
+id                  | Integer | Read       | Bathroom's id.
+--------------------|---------|------------|------------
+name                | [Object](/reference/enums#formats)| Read/Write | **Required**. Bathroom's title, list of locales described in [enums section](/reference/enums#locales).
+bath                | Integer | Read/Write | Bathroom's number of baths.
+shower              | Integer | Read/Write | Bathroom's number of showers.
+wc                  | Integer | Read/Write | Bathroom's number of wc's.
+--------------------|---------|------------|------------
+created_at          | [Time](/reference/enums#formats) | Read       | Bathroom's create time.
+updated_at          | [Time](/reference/enums#formats) | Read       | Bathroom's update time.
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List bathrooms
+
+List all bathrooms for given account.
+
+~~~
+GET /bathrooms
+~~~
+
+<%= render 'json_response', endpoint: "bathrooms",
+  scopes: [{ public: "public-rentals_write" }] %>
+
+## Get a single bathroom
+
+Returns a single bathroom identified by ID.
+
+~~~
+GET /bathrooms/:bathroom_id
+~~~
+
+<%= render 'json_response', endpoint: "bathrooms",
+  scopes: [{ public: "public-rentals_write" }] %>
```

Comment:
Actually its mixed, but most with public define only public scope here, when there is no difference in response.
Leave as it is.